db/schema.rb

```diff
@@ -444,12 +454,15 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "sign_in_count", default: 0, null: false
-    t.datetime "current_sign_in_at", precision: nil
-    t.datetime "last_sign_in_at", precision: nil
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
     t.string "current_sign_in_ip"
     t.string "last_sign_in_ip"
     t.string "trn"
     t.uuid "uuid", default: -> { "gen_random_uuid()" }, null: false
+    t.string "secret_key"
+    t.integer "otp_guesses", default: 0, null: false
+    t.datetime "otp_created_at"
```

Comment:
I'm not sure we want these back, they were removed in https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1626.